Api body

evgenkolesman opened the pull request:
Api based on collections

evgenkolesman committed:
Body application ready and some tests

evgenkolesman committed:
Body application ready and some tests

evgenkolesman committed:
Tests junit + mocks

evgenkolesman committed:
Tests junit + mocks

evgenkolesman committed:
Merge branch 'for_testing' into for_application_body

evgenkolesman committed:
Fixed mistakes

evgenkolesman committed:
Fixed mistakes

evgenkolesman committed:
Fixed mistakes

evgenkolesman committed:
Fixed mistakes

evgenkolesman committed:
Fixed mistakes

evgenkolesman committed:
Merge remote-tracking branch 'origin/for_application_body' into for_application_body

evgenkolesman committed:
fixed mistake

evgenkolesman committed:
activation swagger + some fixes with plugin

evgenkolesman committed:
activation swagger + some fixes with plugin

evgenkolesman committed:
Fixed some statuses and removed validations of personId because it seems to be redundant

evgenkolesman committed:
Fixed some statuses and removed validations of personId because it seems to be redundant

evgenkolesman committed:
Fixed a bag with empty dates + searches + date type fixing + test code format fixes

evgenkolesman committed:
Fixed a bag with empty dates + searches + date type fixing + test code format fixes

mih-kopylov commented on pom.xml:
```diff
@@ -31,10 +31,12 @@
             <artifactId>spring-boot-starter</artifactId>
```
spring-boot-starter is a part of spring-boot-starter-web, could be omitted

evgenkolesman replied:
- [x] fixed

mih-kopylov commented on pom.xml:
```diff
@@ -31,10 +31,12 @@
-            <artifactId>spring-boot-starter-web</artifactId>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>2.0.2</version>
```
versions are better to store in properties

mih-kopylov replied:
but in this case version is provided by spring boot parent

evgenkolesman replied:
- [X] fixed

mih-kopylov commented on src/test/java/ControllersTest/LostPassportControllerTest.java:
```diff
@@ -0,0 +1,92 @@
+package ControllersTest;
```
package naming is not in agreed format, usually it's lowercase and replicate packages from main

evgenkolesman replied:
- [X] Fixed

mih-kopylov commented on pom.xml:
```diff
@@ -72,6 +84,10 @@
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```
it's added by default, no need to declare here if it's not re-configured

evgenkolesman replied:
- [x] replaced

evgenkolesman committed:
Fixing api advices from mentor

evgenkolesman committed:
Fixing api advices from mentor

evgenkolesman committed:
Fix interfaces, endpoints

evgenkolesman committed:
Fix interfaces, endpoints

evgenkolesman committed:
Fixing interfaces, endpoints

evgenkolesman committed:
Fixing interfaces, endpoints

evgenkolesman committed:
Merge branch 'for_swagger' into for_application_body

evgenkolesman committed:
Add interfaces in repos, replace interfaces in service, pom fixes, layer fixes, fix tests, add configuration repository and bean for ModelMaker, add properties for messages

evgenkolesman committed:
Add interfaces in repos, replace interfaces in service, pom fixes, layer fixes, fix tests, add configuration repository and bean for ModelMaker, add properties for messages

evgenkolesman committed:
with logging exceptions and ResponseStatus Exception

Add interfaces in repos, replace interfaces in service, pom fixes, layer fixes, fix tests, add configuration repository and bean for ModelMaker, add properties for messages

evgenkolesman committed:
with logging exceptions and ResponseStatus Exception

Add interfaces in repos, replace interfaces in service, pom fixes, layer fixes, fix tests, add configuration repository and bean for ModelMaker, add properties for messages

evgenkolesman committed:
Version with Exception Handler and Controller advice and Enums (version 2)

evgenkolesman committed:
Version with Exception Handler and Controller advice and Enums (version 2)

evgenkolesman committed:
Merge branch 'for_exception_versions' into for_application_body

# Conflicts:
#	src/main/java/com/sperasoft/passportapi/service/PassportServiceImpl.java
#	src/main/java/com/sperasoft/passportapi/service/PersonServiceImpl.java
#	src/main/java/com/sperasoft/passportapi/service/SearchServiceImpl.java
#	src/test/java/controllerstest/PassportControllerTest.java
#	src/test/java/controllerstest/PersonControllerTest.java
#	src/test/java/controllerstest/SearchControllerTest.java
#	src/test/java/servicetest/PassportServiceTest.java
#	src/test/java/servicetest/PersonServiceTest.java

evgenkolesman committed:
Version with Exception Handler and Controller advice with universal handler version 3(1)

evgenkolesman committed:
Version with Exception Handler and Controller advice with universal handler version 3(1)

evgenkolesman committed:
Rest tests for api

evgenkolesman committed:
Rest tests for api

mih-kopylov commented on pom.xml:
```diff
@@ -54,7 +37,23 @@
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>${rest-assured.version}</version>
+<!--            <scope>test</scope>-->
```
the test dependencies should be with `test` scope

evgenkolesman replied:
- [x]  fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/resttest/PassportRestTest.java:
```diff
@@ -0,0 +1,252 @@
+@Slf4j
+@SpringBootTest(classes = PassportApiApplication.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class PassportRestTest {
```
Let's move tests to `test` directory rather than `main`

evgenkolesman replied:
- [x] moved

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/PersonController.java:
```diff
@@ -0,0 +1,39 @@
+
+    @PostMapping
+    public PersonResponse createPerson(@RequestBody @Valid PersonRequest person) {
+        return personService.addPerson(person);
```
PersonService should not be aware about controller level DTOs.

evgenkolesman replied:
- [x]  fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/PersonController.java:
```diff
@@ -0,0 +1,39 @@
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public PersonResponse deletePerson(@PathVariable String id) {
```
There's no need to return a `PersonResponse` if the status is `204`

evgenkolesman replied:
- [x] fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/PassportController.java:
```diff
@@ -0,0 +1,60 @@
+import java.util.List;
+
+@RestController
+@RequestMapping("/person/{personId}")
```
`/passport` could be here as well for consistency with `PersonController`

evgenkolesman replied:
- [x]  fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PassportServiceImpl.java:
```diff
@@ -0,0 +1,150 @@
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class PassportServiceImpl {
```
Could be just `PassportService` without `Impl`

evgenkolesman replied:
- [x]  renamed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/PassportController.java:
```diff
@@ -0,0 +1,60 @@
+
+    @GetMapping("/passport")
+    public List<PassportResponse> findPersonPassports(@PathVariable String personId,
+                                                      @RequestParam(defaultValue = "") String active,
```
`String active` - should be `Boolean`

evgenkolesman replied:
- [x] all fields types are changed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/controller/PassportController.java:
```diff
@@ -0,0 +1,60 @@
+    private final PassportServiceImpl passportService;
+
+    @GetMapping("/passport")
+    public List<PassportResponse> findPersonPassports(@PathVariable String personId,
```
`@PathVariable` is worth to have with value to declare it explicitly

evgenkolesman replied:
- [x]  all variables are named in special fields

evgenkolesman committed:
First part of Fix:
DateTime -> openAPi
DTO removing from services
FIXING PATH for tests
Fixing Boolean for active
updating tests

commit with LocalDateTime

evgenkolesman committed:
First part of Fix:
DateTime -> openAPi
DTO removing from services
FIXING PATH for tests
Fixing Boolean for active
updating tests

commit with LocalDateTime

evgenkolesman committed:
First part of Fix:
DateTime -> openAPi
DTO removing from services
FIXING PATH for tests
Fixing Boolean for active
updating tests
#2
added Predicate in PassportService
updated ExceptionHandler
with LocalDate
#3
PredicateDatesChecking - predicate Bean
And some little fix

evgenkolesman committed:
First part of Fix:
DateTime -> openAPi
DTO removing from services
FIXING PATH for tests
Fixing Boolean for active
updating tests
#2
added Predicate in PassportService
updated ExceptionHandler
with LocalDate
#3
PredicateDatesChecking - predicate Bean
And some little fix

evgenkolesman committed:
Instant.class replace ZonedDateTime.class

+ qualifier type of bean added (that how we can use different ways of injecting)

Instant.class replace ZonedDateTime.class

+ qualifier type of bean added (that how we can use different ways of injecting)

evgenkolesman committed:
Instant.class replace ZonedDateTime.class

+ qualifier type of bean added (that how we can use different ways of injecting)

Instant.class replace ZonedDateTime.class

+ qualifier type of bean added (that how we can use different ways of injecting)

evgenkolesman committed:
Updates:
1) ConfigCommon common class for all beans(other config became deprecated)
2) Start inserting different profiles for working with different databases(testing and dev)

evgenkolesman committed:
Updates:
1) ConfigCommon common class for all beans(other config became deprecated)
2) Start inserting different profiles for working with different databases(testing and dev)

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/configuration/CommonConfiguration.java:
```diff
@@ -0,0 +1,47 @@
+    private final Environment environment;
+
+    @Bean("EnvConfig")
+    public Environment getEnvironment(){
```
This looks redundant, because it doesn't do anything with a new bean

evgenkolesman replied:
- [x] removed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/configuration/CommonConfiguration.java:
```diff
@@ -0,0 +1,47 @@
+    }
+
+    public static ModelMapper configMapper() {
+        return model;
```
Having a static method that returns a bean look like "code smells" because Controller calls static method of regular DTO, which calls this static method, which again takes a Spring bean.

evgenkolesman replied:
- [x] removed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/configuration/CommonConfiguration.java:
```diff
@@ -0,0 +1,47 @@
+    @Bean
+    public BiPredicate<Passport, List<Instant>> getPassportListBiPredicate() {
+        return (passport, timeList) ->
+                (timeList.get(0).isBefore(passport.getGivenDate())) &&
```
List is not a best container for such things: it could contain from 0 to * elements.

It's worth using guava's Range instead

evgenkolesman replied:
- [x] fixed with guava

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/model/Passport.java:
```diff
@@ -0,0 +1,30 @@
+
+    public static Passport of(PassportRequest passportRequest) {
+        Passport passport = CommonConfiguration.configMapper().map(passportRequest, Passport.class);
+        passport.setId(FriendlyId.createFriendlyId());
```
it's worth having DTOs immutable, so that you can easily use them in Hash collections, and can trust the input data

evgenkolesman replied:
- [x] fixed all fields of models classes with final, updated fabric methods with private constructors

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors + fixed tests + updates with builder UriComponentBuilder was chosen

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors + fixed tests + updates with builder UriComponentBuilder was chosen

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors + fixed tests + updates with builder UriComponentBuilder was chosen

evgenkolesman committed:
Fix JSON, remove configurations, fix constructors + fixed tests + updates with builder UriComponentBuilder was chosen

Commit:
Merge remote-tracking branch 'origin/for_application_body' into for_application_body

# Conflicts:
#	src/test/java/com/sperasoft/passportapi/controller/rest/PassportRestControllerTest.java

Commit:
database configuration + scripts

Commit:
database configuration + scripts

Commit:
Upd constructors

Commit:
Upd constructors

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/model/Person.java:
```diff
@@ -0,0 +1,48 @@
+        this.list = new ArrayList<>();
+    }
+
+    public static Person of(final String id, final PersonRequest personRequest) {
```
`Person` from package `model` should not be aware about controller's DTOs

evgenkolesman replied:
- [x] fixed, replaces to allargs contructor

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PersonService.java:
```diff
@@ -0,0 +1,52 @@
+@RequiredArgsConstructor
+public class PersonService {
+
+    private final PersonRepository personRepositoryImpl;
```
worth renaming to `personRepository`

evgenkolesman replied:
- [x] all was renamed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/SearchService.java:
```diff
@@ -0,0 +1,55 @@
+
+
+    public Person findPersonByPassportNumber(String number) {
+        return personRepositoryImpl.findAll().stream().map(person -> personRepositoryImpl.findAll()
```
`personRepositoryImpl.findAll()` loads all the database table, that might be too much

evgenkolesman replied:
- [x] this method was refactored

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PersonService.java:
```diff
@@ -0,0 +1,52 @@
+                        && person1.getBirthday().isEqual(person.getBirthday())
+                        && person1.getBirthdayCountry().equals(person.getBirthdayCountry())
+        )) {
+           throw new InvalidPersonDataException();
```
This seems not a responsibility of a service, but a repository

evgenkolesman replied:
- [x] replaced to repository

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PersonService.java:
```diff
@@ -0,0 +1,52 @@
+        if (personRepositoryImpl.findById(id) == null) {
+            throw new PersonNotFoundException(id);
+        }
+            return personRepositoryImpl.findById(id);
```
`personRepositoryImpl.findById(id)` is executed twice. Second one seems redundant

evgenkolesman replied:
- [x] fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PersonService.java:
```diff
@@ -0,0 +1,52 @@
+
+    private void checkPersonPresentInRepository(String id) {
+        if (findById(id) == null) {
+            throw new PersonNotFoundException(id);
```
That's also not a responsibility of the service, but repository

evgenkolesman replied:
- [x] fixed

mih-kopylov commented on src/main/java/com/sperasoft/passportapi/service/PersonService.java:
```diff
@@ -0,0 +1,52 @@
+
+    public Person updatePerson(String id, Person person) {
+        checkPersonPresentInRepository(id);
+        return personRepositoryImpl.updatePerson(id, person);
```
what if I create two persons, and with a second request change country for the value from the first one? The uniqueness would be broken

evgenkolesman replied:
- [x] synchronised all crud operations

Commit:
First step updating with new Decisions with connections between models and repository layer logic

Commit:
First step updating with new Decisions with connections between models and repository layer logic

Commit:
Fix recommendations

Commit:
Fix recommendations

Commit:
First step updating with new Decisions with connections between models and repository layer logic

Commit:
First step updating with new Decisions with connections between models and repository layer logic

Commit:
Merge branch 'for_postgres' into for_Docker_DB

# Conflicts:
#	src/main/resources/application.properties

Commit:
Second step adding postgres from docker-compose

Commit:
Second step adding postgres from docker-compose

Commit:
Third step adding PersonRepositoryWithDBImpl.java

Commit:
Third step adding PersonRepositoryWithDBImpl.java

Commit:
fourth step adding all operations from interfaces to DB implementations

Commit:
fourth step adding all operations from interfaces to DB implementations

Commit:
updates with db and flyway

Commit:
updates with db and flyway

Commit:
Merge branch 'for_Docker_DB' into for_application_body

# Conflicts:
#	src/main/java/com/sperasoft/passportapi/controller/PersonController.java
#	src/main/java/com/sperasoft/passportapi/repository/PassportRepositoryImpl.java
#	src/main/java/com/sperasoft/passportapi/repository/PersonRepository.java
#	src/main/java/com/sperasoft/passportapi/repository/PersonRepositoryImpl.java
#	src/main/java/com/sperasoft/passportapi/service/PassportService.java
#	src/main/java/com/sperasoft/passportapi/service/PersonService.java
#	src/test/java/com/sperasoft/passportapi/service/PassportServiceTest.java
#	src/test/java/com/sperasoft/passportapi/service/PersonServiceTest.java

Commit:
Merge Docker for DB

Commit:
Merge Docker for DB

Commit:
try CI/CD

Commit:
try CI/CD

evgenkolesman committed:
Create maven.yml

evgenkolesman committed:
Create maven.yml

Commit:
remove db add CI/CD

Commit:
remove db add CI/CD

Commit:
remove db add CI/CD with 11 JDK and java version

Commit:
remove db add CI/CD with 11 JDK and java version

Commit:
remove db add CI/CD with 17 JDK and java version

Commit:
remove db add CI/CD with 17 JDK and java version

Commit:
first part of refactoring tests
PersonController
PassportController

Commit:
first part of refactoring tests
PersonController
PassportController

evgenkolesman committed:
first part of refactoring tests

evgenkolesman committed:
first part of refactoring tests

evgenkolesman committed:
refactoring tests
fix GIT userName and email
fix Ci/CD

evgenkolesman committed:
refactoring tests
fix GIT userName and email
fix Ci/CD

evgenkolesman committed:
add CI/CD
Need to fix problem with docker-compose running in CI/CD

evgenkolesman committed:
add CI/CD
Need to fix problem with docker-compose running in CI/CD

evgenkolesman committed:
add CI/CD
Need to fix problem with docker-compose running in CI/CD

evgenkolesman committed:
add CI/CD
Need to fix problem with docker-compose running in CI/CD

evgenkolesman committed:
Fixed validations

evgenkolesman committed:
Fixed validations

evgenkolesman committed:
Add badge CI/CD

evgenkolesman committed:
Add badge CI/CD

evgenkolesman committed:
Merge branch 'for_application_body' into for_Docker_DB

# Conflicts:
#	.github/workflows/maven.yml
#	src/main/resources/application.properties
#	src/test/java/com/sperasoft/passportapi/controller/mocks/PassportControllerTest.java
#	src/test/java/com/sperasoft/passportapi/controller/rest/PassportRestControllerTest.java
#	src/test/java/com/sperasoft/passportapi/controller/rest/SearchRestControllerTest.java
#	src/test/java/com/sperasoft/passportapi/service/PassportServiceTest.java

evgenkolesman committed:
Add badge CI/CD

evgenkolesman committed:
Add badge CI/CD

evgenkolesman committed:
Add ErrorModel

evgenkolesman committed:
Add ErrorModel

evgenkolesman committed:
Add ErrorModel

evgenkolesman committed:
Add BD

evgenkolesman committed:
Add BD

evgenkolesman committed:
fixings

evgenkolesman committed:
fixings

evgenkolesman committed:
delete mocks

evgenkolesman committed:
delete mocks

evgenkolesman committed:
updates

evgenkolesman committed:
updates

evgenkolesman committed:
updates

evgenkolesman committed:
updates docker compose volumes

evgenkolesman committed:
updates docker compose volumes

evgenkolesman committed:
updates docker compose volumes

evgenkolesman committed:
Experiment with skipping tests

evgenkolesman committed:
Revert "Experiment with skipping tests"

This reverts commit 92f912eb14009e28a6d67f1e99b22660970d198a.

evgenkolesman committed:
Experiment with skipping tests

evgenkolesman committed:
NEED TO TEST API FULL

evgenkolesman committed:
NEED TO TEST API FULL

evgenkolesman committed:
First step testing

evgenkolesman committed:
First step testing

evgenkolesman committed:
First step testing

evgenkolesman committed:
First step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Second step testing

evgenkolesman committed:
Merge branch 'for_application_body' into for_Docker_DB

# Conflicts:
#	.github/workflows/maven.yml
#	pom.xml
#	src/main/java/com/sperasoft/passportapi/controller/dto/PassportRequest.java
#	src/main/java/com/sperasoft/passportapi/controller/dto/PersonRequest.java
#	src/main/java/com/sperasoft/passportapi/controller/dto/PersonResponse.java
#	src/main/java/com/sperasoft/passportapi/exceptions/RestResponseEntityExceptionHandler.java
#	src/main/java/com/sperasoft/passportapi/model/ErrorModel.java
#	src/main/java/com/sperasoft/passportapi/repository/PersonRepositoryImpl.java
#	src/main/resources/application.properties
#	src/test/java/com/sperasoft/passportapi/controller/PassportRestControllerTest.java
#	src/test/java/com/sperasoft/passportapi/controller/PersonRestControllerTest.java
#	src/test/java/com/sperasoft/passportapi/service/PassportServiceTest.java
#	src/test/java/com/sperasoft/passportapi/service/SearchServiceTest.java
#	src/test/resources/application.properties

evgenkolesman committed:
merging and updating to application body

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
Merge branch 'for_application_body' into for_Docker_DB

# Conflicts:
#	.github/workflows/maven.yml
#	pom.xml
#	src/test/java/com/sperasoft/passportapi/controller/SearchRestControllerTest.java

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
Merge remote-tracking branch 'origin/for_application_body' into for_application_body

# Conflicts:
#	src/test/java/com/sperasoft/passportapi/controller/SearchRestControllerTest.java

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
testing over

evgenkolesman committed:
Merge branch 'for_application_body' into for_Docker_DB

# Conflicts:
#	.github/workflows/maven.yml
#	pom.xml
#	src/test/java/com/sperasoft/passportapi/controller/SearchRestControllerTest.java

evgenkolesman committed:
DB setting up

evgenkolesman committed:
test refactoring

evgenkolesman committed:
test refactoring

evgenkolesman committed:
Merge branch 'for_application_body' into for_Docker_DB

# Conflicts:
#	src/test/java/com/sperasoft/passportapi/controller/PassportRestControllerTest.java
#	src/test/java/com/sperasoft/passportapi/controller/abstracts/pasporttests/PassportCreateTests.java
#	src/test/java/com/sperasoft/passportapi/controller/persontest/PersonUpdateTests.java

evgenkolesman committed:
Merge branch 'for_Docker_DB' into for_application_body

# Conflicts:
#	.github/workflows/maven.yml
#	pom.xml
#	src/main/java/com/sperasoft/passportapi/controller/dto/PassportResponse.java
#	src/main/java/com/sperasoft/passportapi/model/Passport.java
#	src/main/java/com/sperasoft/passportapi/repository/PassportRepository.java
#	src/main/java/com/sperasoft/passportapi/repository/PassportRepositoryImpl.java
#	src/main/java/com/sperasoft/passportapi/service/PersonService.java
#	src/main/resources/application.properties
#	src/main/resources/exceptions.properties
#	src/test/java/com/sperasoft/passportapi/controller/abstracts/PersonTestMethodContainer.java
#	src/test/java/com/sperasoft/passportapi/service/PassportServiceTest.java
#	src/test/resources/application.properties

evgenkolesman committed:
updates
